src/cmd/run.go

```diff
@@ -481,6 +488,10 @@ func constructExecArgs(container string,
 
 	execArgs = append(execArgs, envOptions...)
 
+	if runFlags.preserveFDs != 0 {
```

Comment:
Another thing that I just realized ...

It might be better if `runFlags.preserveFDs` is passed down to `runCommand()` as a parameter, because the code from `runCommand()` downwards is shared between the `enter` and `run` commands.

I am still getting used to Go where nothing is `static` to one file.  Everything is either global within a package or exported from the package.  So, far, it seems like people tend to restrict the parsed flag variables to the topmost function.

Reply:
I took the liberty to change this.